config/kubernetes/config.go

```diff
@@ -0,0 +1,62 @@
+package kubernetes
```

Comment:
```suggestion
package container
```

lets use consistent package names as groups. folder name need to be renamed as well.